src/chromium/crBrowser.ts

```diff
@@ -50,6 +51,8 @@ export class CRBrowser extends BrowserBase {
     const browser = new CRBrowser(connection, options);
     browser._devtools = devtools;
     const session = connection.rootSession;
+    const version = await session.send('Browser.getVersion');
+    browser._isMac = version.userAgent.includes('Macintosh');
```

Comment:
```js
const os = require('os');
const isMac = os.platform() === "darwin"
```
wouldn't it be better to go over the os.platform() instead over the browser and User-Agent to determine the operating system?

Reply:
It would be better! But we can't because the machine running playwright might be a different OS from the machine running the browser.

Reply:
Gotcha, for the RPC client/server architecture 👍